assets/js/translator.js

```diff
@@ -132,6 +133,12 @@ if(modeEnabled('translation')) {
             }, timeout);
 
             synchronizeTextareaHeights();
+
+            if(isURL($('#originalText').val())) {
+                setTimeout(function () {
+                    translateWebpage();
```

Comment:
Is there a reason we need to be setting two timeouts? Seems to be reasonable to only set a single one inside this handler.